packages/loader/container-loader/src/container.ts

```diff
@@ -771,6 +770,9 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             // Propagate current connection state through the system.
             this.propagateConnectionState();
             if (!this.closed) {
+                this._deltaManager.inbound.resume();
```

Comment:
So I guess we don't "block" anything on processing the ops. Is that correct? So when I get the container back from the loader, the ops may not have been applied yet, right?

Reply:
That is correct. I'm looking at it as multi-stage work. First - ensure that ops can trickle through as quickly as possible. This ensures that if ops are coming from storage (not cached in any form), they make it as soon as possible to end result.
The there potential fix - figure out how to guarantee that ops that are cached applied before  container returns.